From d08c0fd5bd0d5c2b128e0057f7441dea750b5114 Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@foss.st.com>
Date: Mon, 15 Jan 2024 11:41:23 +0100
Subject: [PATCH 0788/1141] arm64: dts: st: add sleep pins for Eth1 in
 stm32mp257f-dk

Add analog gpio pin configuration ("sleep") to manage power mode on
stm32mp257 DK board.

Signed-off-by: Christophe Roullier <christophe.roullier@foss.st.com>
Change-Id: I19d1bc7bb13f37c63a71d4a745a50f0bfcb1f5ad
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/351281
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Christophe ROULLIER <christophe.roullier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Christophe ROULLIER <christophe.roullier@st.com>
Domain-Review: Christophe ROULLIER <christophe.roullier@st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -231,7 +231,8 @@
 &eth1 {
 	status = "okay";
 	pinctrl-0 = <&eth1_rgmii_pins_b>;
-	pinctrl-names = "default";
+	pinctrl-1 = <&eth1_rgmii_sleep_pins_b>;
+	pinctrl-names = "default", "sleep";
 	phy-mode = "rgmii-id";
 	max-speed = <1000>;
 	phy-handle = <&phy1_eth1>;
